_docs/_latest/configure/notifier/index.md

```diff
@@ -2,4 +2,133 @@
 title: Notifier
 order: 300
 ---
+
 # {{ page.title }}
+
+Notifier can send a variety of notifications to alert you
+of Forseti events. You can configure notifications to send to different
+channels and in different formats.
+
+### Notification Types
+
+  1. Inventory summary: Count of the resources added in the latest inventory crawl.
```

Comment:
This doesn't need to be a numbered/ordered list.

Reply:
Done